src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -176,6 +179,16 @@ public static function getTermDescr($termid)
         return $return['descr'].date(' Y', strtotime($return['start']));
     }
 
+    public static function getTermWeeks($termid)
```

Comment:
nit: would be good to add a phpdoc comment or at least a return type annotation, so I don't have to look at the schema to find out that `weeks` is an int